pkg/asset/ignition/bootstrap/bootstrap.go

```diff
@@ -218,6 +225,11 @@ func (a *Bootstrap) addStorageFiles(base string, uri string, templateData *boots
 	}
 
 	name := info.Name()
+	if _, ok := proxy[name]; ok {
```

Comment:
If you make `proxy` a `map[string]bool` and populate it with `true` values, then you can right this expression as the more simple `if proxy[name] {`.